src/Middleware/EventDispatcherMiddleware.php

```diff
@@ -20,21 +20,16 @@ class EventDispatcherMiddleware implements MiddlewareInterface
     protected $eventDispatcher;
 
     /** @var array */
-    protected $events;
-
-    /** @var string */
-    protected $clientId;
+    protected $events = [];
```

Comment:
**suggestion:** 
```suggestion
    protected array $events = [];
```

Reply:
As it's a rector change, I won't touch this (might be a BC break _if_ this property is badly used -- maybe this should be in a new PR)